backend/src/main/java/com/chinguv45tier3team53/tripplannerrest/TripPlannerRestApplication.java

```diff
@@ -0,0 +1,18 @@
+package com.chinguv45tier3team53.tripplannerrest;
+
+import com.chinguv45tier3team53.tripplannerrest.entity.Trip;
```

Comment:
there are some unnecessary imports

Reply:
All unused imports will be removed at a later stage since we are still finalizing lots of backend design